ResourceManager/LinkFactory.cs

```diff
@@ -12,12 +12,14 @@ public interface ILinkFactory
         string CreateTagLink(string tag);
         string CreateBranchLink(string noPrefixBranch);
         string CreateCommitLink(ObjectId objectId, string linkText = null, bool preserveGuidInLinkText = false);
+        string CreateShowAllLink(string what);
         string ParseLink(string linkText);
     }
 
     public sealed class LinkFactory : ILinkFactory
     {
         private readonly ConcurrentDictionary<string, string> _linksMap = new ConcurrentDictionary<string, string>();
+        private readonly TranslationString _showAll = new TranslationString("[ Show all ]");
```

Comment:
This `TranslationString` and those of the static class RefsFormatter are not detected.
How should I add them? `ITranslate` seems to be not prepared for `static` fields at first glance.